Annotato/Annotato/Persistence/PersistenceManager/AnnotationsPersistence.swift

```diff
@@ -4,4 +4,5 @@ protocol AnnotationsPersistence {
     func createAnnotation(annotation: Annotation) async -> Annotation?
     func updateAnnotation(annotation: Annotation) async -> Annotation?
     func deleteAnnotation(annotation: Annotation) async -> Annotation?
+    func createOrUpdateAnnotationsForLocal(annotations: [Annotation]) -> [Annotation]?
```

Comment:
The persistence layer does not need to know about whether it's local or not. It's just saying that all persistence classes should implement this method.
```suggestion
    func createOrUpdateAnnotations(annotations: [Annotation]) -> [Annotation]?
```